empress/topoSort/test_ReconBuilder.py

```diff
@@ -0,0 +1,150 @@
+import unittest
+import os
+import itertools
+import shutil
+from empress import newickFormatReader
+from empress.topoSort import ReconBuilder
+from empress.clumpr import DTLReconGraph
+from empress.clumpr import HistogramAlgTools
+from empress.clumpr.script02_gen_newick_trees import generateNewickTestsMultipleSizes
+
+class TestReconBuilder(unittest.TestCase):
+    """
+    Test the build_trees_with_temporal_order function from ReconBuilder
+    """
+    @classmethod
+    def setUp(self):
+        """
+        generate newick tests of certain sizes
+        """
+        size_range = [5]
+        generated_dir_path = "./newickSample"
```

Comment:
I prefer `./temp_newick_samples` as the directory path. The `temp` is there to ensure the name does not match with other folders in the future.